app/presenters/publishing_api/links_presenter.rb

```diff
@@ -37,7 +37,19 @@ def statistical_data_set_ids
     end
 
     def organisation_ids
-      (item.try(:organisations) || []).map(&:content_id)
+      if item.try(:edition_organisations)
+        item
+          .try(:edition_organisations)
+          .sort_by { |organisation| [organisation.lead_ordering ? 0 : 1, organisation.lead_ordering] }
+          .map(&:organisation)
+          .map(&:content_id)
+      elsif item.try(:organisations)
```

Comment:
This is required because statistics announcements don't have `edition_organisations` (they have `statistics_announcement_organisations` instead).